bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/ui/LicensedJettyActivator.java

```diff
@@ -29,30 +26,26 @@
 import org.eclipse.passage.lic.internal.base.logging.Logging;
 import org.eclipse.passage.lic.internal.equinox.EquinoxPassage;
 import org.eclipse.passage.lic.internal.equinox.LicensedApplication;
-import org.eclipse.passage.lic.internal.equinox.io.FileFromBundle;
 import org.eclipse.passage.lic.internal.jetty.JettyHandler;
 import org.eclipse.passage.lic.internal.jetty.JettyServer;
 import org.eclipse.passage.lic.internal.net.connect.Port;
-import org.osgi.framework.Bundle;
 import org.osgi.framework.BundleActivator;
 import org.osgi.framework.BundleContext;
-import org.osgi.framework.FrameworkUtil;
 
-public final class JettyActivator implements BundleActivator {
+public abstract class LicensedJettyActivator implements BundleActivator {
```

Comment:
I don not see where it was removed from the MANIFEST.MF after convertion to `abstract`

Reply:
It's `final` ancestor resides in `lbc.jetty` and is mentioned in `manifest.mf` properly